src/desktop.md

```diff
@@ -5,46 +5,58 @@ toc: true
 ---
 
 Desktop support allows you to compile Flutter source code
-to a native Windows, macOS, or Linux desktop app. Flutter's desktop
-support also extends to plugins&mdash;you can install 
-existing plugins that support the Windows, macOS, or Linux platforms,
-or you can create your own.
+to a native Windows, macOS, or Linux desktop app.
```

Comment:
s/app/application/

Apps are a diminutive for phones, IIUC.

Reply:
Seriously?!?! I have never heard that before.

Reply:
https://www.dictionary.com/browse/app 🤪

Reply:
Here's the Google developer documentation style guide reference:
https://developers.google.com/style/word-list#app

> In general, use _app_ instead of _application_ when referring to programs for end users, especially in the context of mobile or web software. In some contexts, such as enterprise software, it's OK to use application to convey a sense of greater complexity. Use application in standard phrases such as application programming interface.

Personally, I also associate _app_ with mobile.

Reply:
Microsoft refers to them as "apps" these days: https://www.microsoft.com/en-us/store/apps/windows

This doesn't help my case against my kids.